index.html

```diff
@@ -45,24 +46,59 @@
     <!-- Static content -->
     <div style="display: none;">
       <div id="instructions-template">
-        <object class="introduction-demo" data="assets/intro.svg" type="image/svg+xml"></object>
-        <p>Link the dots to find the lock pattern. After every attempt the game will tell you how many dots you got right.<p>
+        <object class="introduction-demo" data="../assets/intro.svg" type="image/svg+xml"></object>
+        <p>#@description_game<p>
         <table>
           <tr>
             <td style="width:1.5em;">&#9679;</td>
-            <td>a dot occurs in the pattern and is in the correct order</td>
+            <td>#@description_white</td>
           </tr>
           <tr>
             <td>&#9675;</td>
-            <td>a dot occurs in the pattern but in the wrong order</td>
+            <td>#@description_black</td>
           </tr>
         </table>
-        <p>The difficulty setting changes the amount of dots to connect. Easy is 4 dots, medium is 5 dots and hard is 6 dots.</p>
-        <p>Good luck!_</p>
-        <p class="small">by <a href="https://twitter.com/mxwllt"><svg class="icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-maxwellito"></use></svg></a> / on <a href="https://github.com/maxwellito/breaklock"><svg class="icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-github"></use></svg></a></p>
+        <p>#@description_difficulty</p>
+        <p>#@label_goodluck</p>
+        <p class="small">#@attribution</p>
       </div>
     </div>
 
+    <!-- language selector -->
+    <div id="lang-selector" class="disabled">
```

Comment:
Do you mind if we move the lang selector to a component like the other bits of the app?

Reply:
It makes sense. I botched it up :)